### PR TITLE
[MRG] update(PP): implement new PP

### PR DESCRIPTION
**Pull request by ronioeschger**
Closes parts of https://app.asana.com/0/1187035237749018/1186660118928177

### CODE REVIEW

File: site/content/german/privacy-v2020-08-19.md [hunk 1]
```diff
@@ -57,11 +57,7 @@ Wir verarbeiten personenbezogene Daten im Einklang mit der EU-DSGVO und dem schw
 
 Wir verarbeiten Ihre Daten für die Dauer, die für den jeweiligen Zweck oder die jeweiligen Zwecke erforderlich ist. Bei länger dauernden Aufbewahrungspflichten aufgrund von gesetzlichen und sonstigen Pflichten, denen wir unterliegen, schränken wir die Verarbeitung entsprechend ein.
 
-{{% /markdown %}}
-
-{{% markdown %}}
-
-### Verarbeitung auf unserer Website (www.skribble.com)
+## Verarbeitung auf unserer Website (www.skribble.com)
```
Comment (Kirastina) [inline site/content/german/privacy-v2020-08-19.md:60]:
```suggestion
### Verarbeitung auf unserer Website (www.skribble.com)
```

**Comment by Kirastina** [comment]
To merge on August 19th 2020 (after double checking)

**Comment by Kirastina** [comment]
To merge on August 19th 2020 (after double checking)

**Comment by dkumpf1** [comment]
[DEPLOY-PREVIEW]

**Tested: OK**

I tested this and it looks good from my point of view.

**Pull request merged**